README.rst

```diff
@@ -29,7 +29,23 @@ The basic idea for calculating estimations is the following:
 
 1. `SPRINT_HOURS_RESERVED_FOR_MEETINGS` hours are reserved for the meetings for each sprint.
 2. `SPRINT_HOURS_RESERVED_FOR_EPIC_MANAGEMENT` hours are reserved for epic management for each sprint.
-3. 1 hour is planned for reviewing each task with <= 3 story points. For bigger tasks, 2 hours are reserved.
+3. Reserved time for reviewing is defined in `SPRINT_HOURS_RESERVED_FOR_REVIEW` as a json dict, where you can define the review time for any
+   amount of story points, and also if no story points are defined("null") for a Jira issue. e.g.
+   .. code::
+
+        SPRINT_HOURS_RESERVED_FOR_REVIEW {
+            "null": 2,
+            "1.9": 1,
+            "2": 3,
+            "5.1": 6
+        }
+
+   Here we are defining that if no story point is set for an issue, it will reserve 2 hours, for 1.9 story points it will reserve 1 hour,
```

Comment:
```suggestion
   Here we are defining that:

   1.  If no story points are set for an issue, it will reserve 2 hours.
   2.  For 1.9 story points it will reserve 1 hour.
```

Etc…
It will be more readable.